src/styles/blocks/catalog.scss

```diff
@@ -18,8 +18,15 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: 300ms;
```

Comment:
Maybe it is better to make a variable for time transition as you use it twice?

Reply:
fixed it )